roles/tower/tasks/initialize.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Create Tower super user via Django if it doesn't exist.
   shell: >-
-    kubectl exec -n {{ meta.namespace }} {{ tower_pod_name }} -- bash -c
+    kubectl exec -n {{ meta.namespace }} -c tower-task {{ tower_pod_name }} -- bash -c
```

Comment:
I _think_ this will be `awx-task` on upstream deployments.

Reply:
If that's the case (and in CI, we test with `awx` since it's difficult to automate the licensing?), should the container name be a variable?

Reply:
Yeah, I think we can use `meta.name` here too.

Reply:
I don't think `meta.name` will work here. This is the actual container that we need to run again. `meta.name` will just be `tower`